tests/components/core/test_html.py

```diff
@@ -15,8 +15,8 @@ def test_html_many_children():
 
 def test_html_create():
     html = Html.create("<p>Hello !</p>")
```

Comment:
lets add another test case that passes an f-string to `Html.create` to hit https://github.com/reflex-dev/reflex/issues/2155